src/configuration.ts

```diff
@@ -6,6 +6,7 @@ export interface Configuration {
   refreshPullRequestStatus: number;
   remoteName: string;
   upstream?: string;
+  customPullRequestTitle: boolean;
```

Comment:
The `package.json` configuration for the extensions as required by vscode is missing.
Please add it.

Reply:
Oh whoops, new to the whole extension thing. I'll add that right now 👍